### PR TITLE
Overall edit of PyFluent-Visualization doc

### PR DESCRIPTION
**Pull request by PipKat**
This PR has general edits for the PyFluent-Visualization doc. Note, however, that [issue #86](https://github.com/pyansys/pyfluent-visualization/issues/86) must be addressed before I can thoroughly review this documentation. A lot of the guide's existing content is not specific to PyFluent-Visualization but rather to PyFluent-Core. For the current example, I added headings. Our approach is to have a brief heading for best display in the doc's right navigation pane and then have more descriptive text under the heading.

### CODE REVIEW

**Comment by PipKat** [comment]
@jorgepiloto @RobPasMue Can you look at the log for the failed Documentation check. I can't figure out what the error means, let alone fix it!

File: doc/source/api/visualization/matplot_windows_manager.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@ every time step.
     plot1.y_axis_function = "temperature"
     
     
-    #Plot xy plot on window-1.
+    #Plot XY plot on window-1.
```
Comment (jorgepiloto) [inline doc/source/api/visualization/matplot_windows_manager.rst:27]: A white space needs to be left between the comment sign and the comment itself:

```suggestion
    # Plot XY plot on window-1.
```

Comment (jorgepiloto) [reply doc/source/api/visualization/matplot_windows_manager.rst:27]: I will implement a linter for this via blacken-docs tool.

File: doc/source/users_guide/postprocessing.rst [hunk 2]
```diff
@@ -90,13 +90,13 @@ The following example demonstrates how you can display the vector object:
...
-------------------
+Plot data
+---------
 The following plotting operations are supported.
 
-Displaying XY plots
-~~~~~~~~~~~~~~~~~~~
-The following example demonstrates how you can display the XY plot:
+Generate XY plot
+~~~~~~~~~~~~~~~
```
Comment (jorgepiloto) [inline doc/source/users_guide/postprocessing.rst:98]:
```suggestion
Generate XY plot
~~~~~~~~~~~~~~~~
```

**Review by jorgepiloto — CHANGES_REQUESTED** [review]
This may be the reason of the failure, @PipKat.

**Comment by jorgepiloto** [comment]
@PipKat, the documentation is failing because these warnings:

```text
/home/ansys/actions-runner/_work/pyfluent-visualization/pyfluent-visualization/doc/source/index.rst:65: WARNING: undefined label: api reference
/home/ansys/actions-runner/_work/pyfluent-visualization/pyfluent-visualization/doc/source/index.rst:65: WARNING: undefined label: examples
```

which are caused by these lines in the `doc/source/index.rst` file

```text
In addition to installation and usage information, the PyFluent-Visualization
documentation provides :ref:`API reference`, :ref:`Examples`, and :ref:`Contributing`
sections.
```

**Comment by jorgepiloto** [comment]
I fixed the issues in https://github.com/pyansys/pyfluent-visualization/pull/92/commits/5ba507929f6ea217ab832e987ffed67a6c97ca0b and https://github.com/pyansys/pyfluent-visualization/pull/92/commits/782e53c5dec64180434a5d184708ce25e4de3e17, @PipKat.

The "autosectionlabel" extension needed to be enabled and there was a missing ">" at the end of a link. It took me a bit to spot this last error...

**Comment by PipKat** [comment]
Thanks so much for figuring out these problems. Words are hard--but code is nearly impossible!

**Comment by RobPasMue** [comment]
Thanks for looking into this @jorgepiloto!

File: doc/source/users_guide/postprocessing.rst [hunk 3]
```diff
@@ -61,9 +61,9 @@ The following example demonstrates how you can display the iso-surface:
     iso_surf1.iso_value = -0.125017
     surf_outlet_plane.display("window-2")
 
-Displaying contours
-~~~~~~~~~~~~~~~~~~~
-The following example demonstrates how you can display the contour object:
+Display contour
+~~~~~~~~~~~~~~~
+This example demonstrates how you can display the contour object:
```
Comment (ajain-work) [inline doc/source/users_guide/postprocessing.rst:66]:
```suggestion
This example shows how you can display the contour object:
```

File: examples/00-postprocessing/post_processing_exhaust_manifold.py [hunk 4]
```diff
@@ -163,40 +190,58 @@
...
+# Create a vector on a predefined surface.
 
 velocity_vector = graphics.Vectors["velocity-vector"]
 velocity_vector.surfaces_list = ["solid_up:1:830"]
 velocity_vector.scale = 2
 velocity_vector.display("window-8")
 
 ###############################################################################
-# Start the Plot Object for the session
+# Start plot object
```
Comment (ajain-work) [inline examples/00-postprocessing/post_processing_exhaust_manifold.py:203]:
```suggestion
# Create plot object
```

File: examples/00-postprocessing/post_processing_exhaust_manifold.py [hunk 5]
```diff
@@ -163,40 +190,58 @@
...
+# ~~~~~~~~~~~~~~~~~
+# Start the plot object for the session.
```
Comment (ajain-work) [inline examples/00-postprocessing/post_processing_exhaust_manifold.py:205]:
```suggestion
# Create the plot object for the session.
```

File: examples/00-postprocessing/post_processing_exhaust_manifold.py [hunk 6]
```diff
@@ -163,40 +190,58 @@
...
+# Set plot surface and Y-axis function
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Set the surface on which to generate the plot and the Y-axis function.
+
 xy_plot.surfaces_list = ["outlet"]
 xy_plot.y_axis_function = "temperature"
 
 ###############################################################################
-# Plot the created XY-Plot
+# Plot XY plot
```
Comment (ajain-work) [inline examples/00-postprocessing/post_processing_exhaust_manifold.py:225]: Generate?

Comment (RobPasMue) [reply examples/00-postprocessing/post_processing_exhaust_manifold.py:225]: Agreed

Comment (RobPasMue) [inline examples/00-postprocessing/post_processing_exhaust_manifold.py:225]:
```suggestion
# Generate XY plot
```

File: examples/00-postprocessing/post_processing_exhaust_manifold.py [hunk 7]
```diff
@@ -163,40 +190,58 @@
...
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Set the surface on which to generate the plot and the Y-axis function.
+
 xy_plot.surfaces_list = ["outlet"]
 xy_plot.y_axis_function = "temperature"
 
 ###############################################################################
-# Plot the created XY-Plot
+# Generate XY plot
+# ~~~~~~~~~~~~
```
Comment (jorgepiloto) [inline examples/00-postprocessing/post_processing_exhaust_manifold.py:226]:
```suggestion
# Generate XY plot
# ~~~~~~~~~~~~~~~~
```

File: examples/00-postprocessing/post_processing_exhaust_manifold.py [hunk 8]
```diff
@@ -163,40 +190,58 @@
...
 
 velocity_vector = graphics.Vectors["velocity-vector"]
 velocity_vector.surfaces_list = ["solid_up:1:830"]
 velocity_vector.scale = 2
 velocity_vector.display("window-8")
 
 ###############################################################################
-# Start the Plot Object for the session
+# Create plot object
+# ~~~~~~~~~~~~~~~~~
```
Comment (jorgepiloto) [inline examples/00-postprocessing/post_processing_exhaust_manifold.py:204]:
```suggestion
# Create plot object
# ~~~~~~~~~~~~~~~~~~
```

**Review by jorgepiloto — CHANGES_REQUESTED** [review]

**Review by RobPasMue — APPROVED** [review]

**Review by dnwillia-work — APPROVED** [review]

**Review by jorgepiloto — APPROVED** [review]

**Pull request merged**